build/test-integration.sh

```diff
@@ -55,4 +57,13 @@ dotnet test \
     $testCaseFilter \
     --test-adapter-path:. \
     --logger:"xunit;LogFilePath=$ARTIFACTS_DIR/testResults/$testProjectName$integrationTestPlatform.Tests.xml" \
-    -c $BUILD_CONFIGURATION
\ No newline at end of file
+    -c $BUILD_CONFIGURATION
+
+if [ "$testCaseFilter"=="db" ];then
+		echo "Running integration tests with no categories (tests missed to be tagged in their categories)..."
```

Comment:
This line could be simplified, e.g. "Running uncategorized tests".

Also - point to consider - do we even **want** to have such tests? Maybe the "right thing" to do is to fail the build if **there are any** uncategorized tests? @gpcastro @kichalla